test/roots.jl

```diff
@@ -55,14 +55,19 @@ newtonlike_methods = [Newton, Krawczyk]
 end
 
 
+in_solution_set(point, solution_intervals) =
+    any(map(interval -> point in interval, solution_intervals))
```

Comment:
I believe you remove the `map` and just write `any(interval -> point in interval, solution_intervals)`, which should even be slightly more efficient.

Reply:
Yes, that's correct. I've adjusted the PR.